src/views/Home/components/CLWCreditsDaysLeft/index.js

```diff
@@ -107,11 +105,15 @@ const CLWCreditsDaysLeft = () => {
               {`${daysRemaining} Days Left`}
             </Typography>
           </Grid>
-          <Grid item>
-            <Typography variant="body2" style={{ color: 'gray', textAlign: 'center' }}>
-              {`${remaining} CL&W Credit` + (remaining === 1 ? '' : 's') + ' Left'}
-            </Typography>
-          </Grid>
+          {required !== 0 && current !== 0 ? (
```

Comment:
I added 'current' as well as I was imagining a use case where a student completes their required CL&W credit for a semester, and then can't view their current credits because "required === 0". This way, it only disables when both required and current are 0, which should only occur during summer/winter term when CL&W is not-applicable.

Reply:
You are correct, I got required mixed with remaining. I'll revert it